lib/commands/general.js

```diff
@@ -3,30 +3,37 @@ import fs from 'fs';
 import request from 'request-promise';
 import log from '../logger';
 import Jimp from 'jimp';
-import { getAuthDigestParts, getAuthHeader } from '../utils';
+import {getAuthDigestParts, getAuthHeader} from '../utils';
 
 // Note: it seems like axis's FormData does not work well against
 // the device for now. It got 411 error in the axios's general way
 // to send formData to the target, They required content-length.
 
-async function makePluginReq ({
+/**
+ * @template [TRes=any]
+ * @param {MakePluginReqOpts} opts
+ * @returns {Promise<TRes>}
+ */
+async function makePluginReq({
   opts,
   formData,
   method = 'POST',
   uri = '/plugin_install',
   contentType = 'multipart/form-data',
-  encoding
+  encoding,
 }) {
+  // @ts-expect-error
```

Comment:
@jlipps are these (`rokuHost`, `rokuWebPort`, etc.) caps that need to be defined?

Reply:
yeah, and they're documented as such in the readme. guess they didn't make it into the schema.